package.json

```diff
@@ -1,145 +1,26 @@
 {
-  "name": "@itwin/itwinui-react",
-  "version": "1.34.2",
-  "author": "Bentley Systems",
-  "license": "MIT",
-  "main": "cjs/index.js",
-  "module": "esm/index.js",
-  "typings": "cjs/index.d.ts",
-  "description": "A react implementation of our iTwinUI UX standards",
-  "homepage": "https://github.com/iTwin/iTwinUI-react",
-  "keywords": [
-    "component",
-    "components",
-    "design-system",
-    "design",
-    "framework",
-    "frontend",
-    "itwin",
-    "itwinui",
-    "react-component",
-    "react",
-    "ui",
-    "ux"
+  "name": "turborepo-basic-shared",
+  "version": "0.0.0",
+  "private": true,
+  "workspaces": [
+    "apps/*",
+    "packages/*"
   ],
   "scripts": {
-    "build": "yarn clean && yarn mkdirp lib/ && yarn copy-files && tsc -p tsconfig.cjs.json && tsc -p tsconfig.esm.json",
-    "build:watch": "yarn clean && yarn mkdirp lib/ && yarn copy-files && concurrently \"tsc -p tsconfig.cjs.json --watch\" \"tsc -p tsconfig.esm.json --watch\"",
-    "clean": "yarn clean:coverage && yarn clean:lib",
-    "clean:coverage": "rimraf coverage",
-    "clean:lib": "rimraf lib",
-    "test": "jest",
-    "test:watch": "jest --watch",
-    "createComponent": "node scripts/createComponent.js",
-    "pretty": "prettier --config .prettierrc \"scripts/**/*\" src/{__tests__,core,style,utils}/**/*{tsx,ts,jsx,js,css,scss} --write",
-    "lint": "eslint */**/*.{js,ts,tsx} --max-warnings=0",
-    "lint:fix": "yarn lint --fix && yarn lint:copyright --fix */**/*.{js,ts,tsx}",
-    "lint:copyright": "node scripts/copyrightLinter.js",
-    "copy-files": "cpx package.json lib && cpx \"{README,LICENSE,CHANGELOG}.md \" lib",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
-  },
-  "dependencies": {
-    "@itwin/itwinui-css": "^0.53.2",
-    "@itwin/itwinui-icons-react": "^1.5.0",
-    "@itwin/itwinui-illustrations-react": "^1.0.1",
-    "@tippyjs/react": "^4.2.5",
-    "@types/react-table": "^7.0.18",
-    "classnames": "^2.2.6",
-    "react-table": "^7.1.0",
-    "react-transition-group": "^4.1.1"
+    "build": "turbo run build",
```

Comment:
well ok.. idk anymore..
I ran storybook from react package, seemed ok.
Then i ran again `yarn dev` and now it gave no errors :/ I still have all those prettier changes too, so not sure what happened here

Reply:
`yarn dev` should be fixed now.

Those errors were because the `@itwin/itwinui-react` package needs to be built before it can be used in the vite playground. So I added `^build` dependency and removed the `--parallel` flag.

Storybook should not be affected, I think it might have been because you had some other storybook running somewhere else.